src/schemas/paths.json

```diff
@@ -7,7 +7,7 @@
       "format": "asl_path"
     },
     "asl_ref_path": {
-      "type": "string",
+      "type": ["string", "null"],
       "format": "asl_ref_path"
     },
```

Comment:
This looks good for the ResultPath but I think you'd also need to change the `asl_path` type above this. This is the type that's used for InputPath and OutputPath.

I would expect your test file "valid-null-input.json" would report an error until the "asl_path" also allows for null

Reply:
Huh, I was messing with the files after the fact and pushed the wrong version— I had meant to test `InputPath: null` be invalid and I was making sure I was understanding what I was doing with `asl_ref_path`, but I just checked in the AWS console to be sure, and `InputPath: null` is valid, which makes sense once I think about it a little further.